ls1intum/Artemis
# `General`: Sort courses alphabetically by title

### Checklist
#### General
#### Client

### Motivation and Context
This pull request closes #4239. Courses on course overview and management dashboard are sorted alphabetically ascending by their title.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Management
3. Verify that courses within the same semester are sorted correctly.
4. Navigate to Course Overview
5. Click '# Register for a course'
6. Verify that courses are sorted correctly.

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
![image](https://user-images.githubusercontent.com/73833780/139944685-d40e2ee2-4ef3-43c7-b2b1-52c3413a196b.png)
![image](https://user-images.githubusercontent.com/73833780/139944736-ab9659ca-e8d5-418d-b9d6-dfdb12ade0b1.png)
![image](https://user-images.githubusercontent.com/73833780/139945541-3a511ff9-a736-45fa-952b-a29e0c955e44.png)
